lib/terraform/client.rb

```diff
@@ -50,8 +50,8 @@ def output(name:)
     end
 
     def version
-      execute command: factory.version_command do |value|
-        return ::Terraform::Version.create value: value
+      @version ||= execute command: factory.version_command do |value|
```

Comment:
Looking at https://github.com/bbatsov/ruby-style-guide#no-class-vars class vars are recommended against. Generally I think it would make sense to reuse the client instance here instead, not sure if I have time to dig into it much more and see if it makes sense here or not. Wondering what you think and if we should ignore the no class vars recommendation (comes up on rubocop currently).